examples/parametric/plot_equivalent_width.py

```diff
@@ -155,6 +156,13 @@ def equivalent_width(grids, uv_index, index_window, blue_window, red_window):
             s=10,
         )
         plt.semilogx(grid.metallicity, EqW, linewidth=0.75, color="grey")
+        
+        csv_file = 'synthetic_ew.csv'
+        
+        with open(csv_file, mode='w', newline='') as file:

```

Comment:
why does this example now save a csv of the equivalent widths?

Reply:
Apologies for this, for some reason when I ran `git pull` this script somehow reverted to an older version. Will merge with the current version with the next PR.